src/app/pages/LiquidityMining/components/MiningPool/PoolChart.tsx

```diff
@@ -1,312 +1,123 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 
+import { translations } from 'locales/i18n';
+import { currentNetwork } from 'utils/classifiers';
 import type { LiquidityPool } from 'utils/models/liquidity-pool';
 import ComparisonChart from 'app/components/FinanceV2Components/ComparisonChart';
 import { getAssetColor } from 'app/components/FinanceV2Components/utils/getAssetColor';
 
+type ChartDatum = [number, number]; //[unix timestamp, value]
+type ChartData = ChartDatum[];
+
+export type PoolData = {
```

Comment:
You wouldn't need to export this type since you don't use it anywhere else but I'd suggest you something else - create a `types.ts` file in the `MiningPool` folder and move there all of the type definitions from this file (except `Props` interface).